src/stock-operations/add-stock-operation/stock-items-addition.component.tsx

```diff
@@ -96,6 +98,13 @@ const StockItemsAddition: React.FC<StockItemsAdditionProps> = ({
     control,
   });
 
+  useEffect(() => {
+    if (formContext?.stockItems) {
+      const stockItems = formContext?.stockItems as Array<any>;
```

Comment:
Can we fix the type here instead of any? what do you think @PatrickWaweru

Reply:
You are right. <any> has been replaced by <StockOperationItemDTO>